css/style.css

```diff
@@ -1,3 +1,29 @@
+body {
+	text-align:center;
+}
+
 h1 {
   color: purple;
 }
+
+.box {
+	height: 150px;
+	width:150px;
```

Comment:
Small nitpick - make sure syntax is consistent throughout CSS (i.e. property:*space*value)